### DISCUSSION TITLE
Possibly add OGG functionality(?)

### DISCUSSION THREAD

**Discussion by KVonGit** [textadventures/quest#1381] [Developing Quest]
`play sound` will play an OGG ***if*** you change the extension from **.ogg** to **.wav**.

...after altering the source code a bit at first, of course.

...but I couldn't find a way to make things work without telling Quest an OGG was a WAV. I barely understand how it works in the WebPlayer app, and I really don't understand how it works in the desktop player as far as this is concerned. I think it all comes down to WMP in the desktop player, though.

After a little research, it seems Windows 10 does not support OGG natively. (I always install VLC; so, my tests work, where others (with no Vorbis codecs installed somehow) fail.)

**Comment by vurt834** [comment]
It'd be nice if there was a way to play more than one sound at a time. That's mostly why I used audio tags with ogg files...i would use ogg for the looping music, and the innate play sound command for sound effects because you could layer them together.

**Reply by KVonGit** [reply]
Yeah... It would be nice. I think the issue with that is that the desktop player is using WMP. [^1]

Also, back when I decided to try OGG for the background audio, the desktop player's browser was too old to handle MP3.  I need to redo this test using MP3. If that works now (and I think it will), then we won't need OGG for anything and there will be no Safari browser compatibility to worry about.

Also also, I failed to include it in any of the recent tests, assuming `play sound` should still work on top of HTML audio to play more than one sound at a time.

I'll switch the OGG to MP3, add a second sound, and post the results somewhere, just for completion.

[^1]: I made this a Discussion rather than an Issue because the next version of Quest (Quest Viva) will not be a Windows app and shouldn't have this issue.

**Reply by KVonGit** [reply]
https://github.com/textadventures/quest/discussions/1426#discussioncomment-12106107

The desktop player can handle MP3 in an HTML audio element, as of Quest 5.9. 👍 👍
